actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -63,10 +63,11 @@ def initialize(routes)
         @routes = routes
       end
 
-      def format(formatter, filter = nil)
-        routes_to_display = filter_routes(filter)
+      def format(formatter, **options)
```

Comment:
I know that's YAGNI but I want to add a few more options, for example `--model` and `--exclude`